reviews/templates/reviews/vue_templates/review_list.html

```diff
@@ -118,6 +118,15 @@ <h4>
                 {{ review.get_stage_display }}
                 {% endverbatim %}
             </div>
+            {# For some reason context.review.REVISION and other class variables of continuation are None?#}
+            {# Only context.review.GO returns 0 for me, as it should #}
+            {# Which is why I've hardcoded the 1 here ... #}
```

Comment:
I can answer this one for you. This is a Vue template (and a first-generation one, which is a bit crap compared to what we're migrating to).

Thus, the context used here is not the same as what you'd expect. It's a manually created dict in `reviews/api/views.py`. 
`GO` is in the dict; `REVISION` isn't.  If you add it to the dict, you can reference it here.